src/openvpn/options.c

```diff
@@ -851,7 +851,11 @@ init_options(struct options *o, const bool init_gc)
     o->scheduled_exit_interval = 5;
 #endif
 #ifdef ENABLE_CRYPTO
+#ifdef ENABLE_CRYPTO_WOLFSSL
+    o->ciphername = "AES-256-CBC";
+#else
     o->ciphername = "BF-CBC";
+#endif
```

Comment:
This is still a no go like I explained in my email to the first patch version

Reply:
to be clear, erroring out if no cipher is set would be okay but changing behaviour is not.

Reply:
Hi, I proposed a solution to this in #134.